src/main/java/net/floodlightcontroller/core/internal/OFChannelHandler.java

```diff
@@ -687,7 +688,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause)
 
 	@Override
 	public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
-		log.debug("channelIdle on OFChannelHandler {}", String.format("%08x", System.identityHashCode(this)));
+		//log.debug("channelIdle on OFChannelHandler {}", String.format("%08x", System.identityHashCode(this)));
```

Comment:
@tulioalberton let's leave this debug message in there. If you want to change the log level to not show this, you can adjust the logback.xml file.